aspnetcore/zone-pivot-groups.yml

```
@@ -0,0 +1,12 @@
+# YamlMime:ZonePivotGroups
+groups:
+- id: tooling
```

Since this could be used in different articles, I'd probably give a more generic name such as 
```suggestion
- id: operating-systems
```